PcAtChipsetPkg/Library/AcpiTimerLib/AcpiTimerLib.c

```diff
@@ -13,6 +13,7 @@
 #include <Library/PciLib.h>
 #include <Library/IoLib.h>
 #include <Library/DebugLib.h>
+#include <Uefi/UefiBaseType.h>

```

Comment:
How is this change to this file related.  It is in a different package.

Reply:
The library added the ACPI header, and caused the PcAtChipset build to fail because EFI_GUID was undefined as UefiBaseType.h was not included.

Reply:
It is the ACPI include files that have added EFI_GUID.  Should that be GUID or EFI_GUID?  If it is GUID then only Base.h is required.

Reply:
Good point. I can change it to GUID, otherwise will include the relevant headers within the ACPI includes.